### ISSUE TITLE
Previous code Uncaught TypeError

### ISSUE THREAD

**Issue by juanamm** [gsavio/player-shoutcast-html5#14]
As I mentioned in the previous "Issue" the updated code does not work, it does not connect to the streaming server.
The above code worked until yesterday, now it doesn't show metadata.
**ERROR:**
`Uncaught TypeError: Cannot read property 'replace' of undefined
    at XMLHttpRequest.xhttp.onreadystatechange (script.js:372)`
**LINE 372 script.js**
`let song = data.currentSong.replace(/&apos;/g, '\'');`

**EDIT:**
This problem is because PHP or cURL fails on the side of my hosting.

**Comment by gsavio** [comment]
It is no problem to help. And as I answered in another issue, try to update your project and see if it solves your problem. 

Best regards.

**Comment by juanamm** [comment]
> It is no problem to help. And as I answered in another issue, try to update your project and see if it solves your problem.
> 
> Best regards.

I have updated the project and a new problem has arisen with CORB.
I have created another thread commenting on the problem.
This thread belongs to the previous version, if you want you can close it.

Thank you

**Comment by juanamm** [comment]
Hi Guilherme,
As I mentioned in the new thread, I have cloned the project again and now
the radio server is blocked by CORB.
If you want any other additional information to those that I gave you
through GitHub, tell me and I will share them here.
I wait for your friendly response.
Greetings,
Juan

**Comment by brluk** [comment]
Hi guys! The same issue reported by @juanamm ([##](https://github.com/gsavio/player-shoutcast-html5/issues/14#issue-645785512)) continued even after updating the project. 

**JS Script: LINE 362**
> let song = data.currentSong.replace(/&apos;/g, '\'');

**Comment by juanamm** [comment]
> Hi guys! The same issue reported by @juanamm ([##](https://github.com/gsavio/player-shoutcast-html5/issues/14#issue-645785512)) continued even after updating the project.
> 
> **JS Script: LINE 362**
> 
> > let song = data.currentSong.replace(/'/g, ''');


This problem is because PHP or cURL fails on the side of your hosting.
I have already answered you in the other issue.
